specs/auth/spec.yaml

```diff
@@ -0,0 +1,32 @@
+openapi: 3.0.0
```

Comment:
Not convinced this file should be in this project. The "servers" defined are specific to the EWB server. At the very most it should be a template other projects can pull in to add to their own swagger specs.

Reply:
Yeaup I just put it there thinking the `evolve-conn-jvm` lib was similar to the `ewb-network-routes` but its more reusable I guess. Will change it to a template like you suggested and have the api spec under ewb server